slave/process-fs-home/bin/process-fs_home.sh

```diff
@@ -67,25 +69,28 @@ function process {
 		done
 	fi
 
-
-
-
 	# lines contains homeMountPoint\tlogin\tUID\tGID\t...
 	while IFS=`echo -e "\t"` read U_HOME_MNT_POINT U_LOGNAME U_UID U_GID SOFT_QUOTA_DATA HARD_QUOTA_DATA SOFT_QUOTA_FILES HARD_QUOTA_FILES USER_STATUS USER_GROUPS REST_OF_LINE; do
 		HOME_DIR="${U_HOME_MNT_POINT}/${U_LOGNAME}"
+		TMP_DIR=`mktemp -d --tmpdir="${U_HOME_MNT_POINT}" "tmp-perun-fs_home-${U_LOGNAME}.XXXX"`
+		if [ "$?" -ne 0 ]; then 
+			log_msg  E_CANNOT_CREATE_TEMP
+		fi
+		#set this temp directory for remove when script ends (if still exists)
+		add_on_exit "rm -rf ${TMP_DIR}"
```

Comment:
This command is executed even when the whole slave script finishes successfully, therefore it will write message that it cannot remove TMP_DIR each time.

Reply:
Manual for rm says "-f, --force ignore nonexistent files and arguments, never prompt" so it will ignore non-existent files and directories.